sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: v2
```

Comment:
Nit: this file is supposed to be in `/eng/mgmt/mgmtmetadta`, otherwise LGTM

Reply:
Got it, fixed.

Reply:
@isra-fel please take a look if everything is ok :)

Reply:
Sorry I didn't make this clear. The correct path is: https://github.com/Azure/azure-sdk-for-net/blob/master/eng/mgmt/mgmtmetadata/sql_resource-manager.txt
This file should be overwritten every time you regenerate SDK

Reply:
So sorry about that. I mixed up two branches :)